src/api/db/data/20200421121610_backfill_notified_projects.rb

```diff
@@ -0,0 +1,13 @@
+class BackfillNotifiedProjects < ActiveRecord::Migration[6.0]
+  def up
+    Notification.where.not(notifiable_id: nil, notifiable_type: nil).find_each do |notification|
```

Comment:
if we validate `notifiable_id` and `notifiable_type` we don't have to add this `where.not` here right?

Reply:
discussed with Saray, I'm fine with that!